dev-requirements.txt

```diff
@@ -1,4 +1,3 @@
--e git+https://github.com/cloudify-cosmo/cloudify-dsl-parser@master#egg=cloudify-dsl-parser==3.2a1
--e git+https://github.com/cloudify-cosmo/cloudify-rest-client@master#egg=cloudify-rest-client==3.2a1
--e git+https://github.com/cloudify-cosmo/cloudify-plugins-common@master#egg=cloudify-plugins-common==3.2a1
-nose
```

Comment:
nose actually comes in handy in dev-reqs files when developing. That way, you're not forced to use tox.